MdePkg/Test/Mock/Include/GoogleTest/Protocol/MockHash2.h

```diff
@@ -5,8 +5,8 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
-#ifndef MOCK_HASH2_H
-#define MOCK_HASH2_H
+#ifndef MOCK_HASH2_H_
+#define MOCK_HASH2_H_

```

Comment:
why is there a trailing `_` ?

Reply:
Just noticing that it's true of other mocks like the one of boot services. I'm curious of that's intended for all mocks, I may have missed that.